### DISCUSSION TITLE
Possible to lower brightness even further?

### DISCUSSION THREAD

**Discussion by tomasz1986** [Tyrrrz/LightBulb#329] [Q&A] [answered]
From my testing, the brightness keeps going down until ~50% but then it stays at the same level even when the setting is at a lower value. The same thing happens on multiple devices, so I don't think this is hardware-dependent. It could be related to Windows itself which is the same configuration of Windows 10 on all of them.

However, other programs, e.g. https://github.com/datbnh/ScreenDimmer, are able to make the screen go even completely black when set to 0%. I'm wondering whether it could be possible to make LightBulb also dim the screen further down using a similar technique.

The reason why I need to lower the brightness more than allowed by LightBulb (and the OS) is that I'm using it on a Windows tablet which I need to be able to use at night with no lights turned on. Without additional dimming, the screen is still much too bright even at the minimum brightness.

**Comment by Tyrrrz** [comment, marked as answer]
I'm able to dim mine pretty much to black. Did you unlock the gamma ranges on your PC? There should have been a popup when you run LightBulb.

**Reply by tomasz1986** [reply]
Hmm, so this is strange. I always unlock the gamma ranges when running LightBulb for the first time, but I checked the registry, and the value wasn't there. I've added it manually, and now the brightness can be decreased much further than before.

I had to do this on two systems, while on yet another one, the value was still present in the registry. I wonder what could cause it to get removed from there.

**Reply by Tyrrrz** [reply]
Hard to say. But LightBulb should check that value every time it runs, not just the first time.

**Reply by tomasz1986** [reply]
I don't think it does, or at least something isn't working right, because I've just restarted LightBulb with the whole `GdiICMGammaRange` removed from the registry, and there was no prompt about having to unlock the gamma ranges.

**Reply by Tyrrrz** [reply]
Yeah that would imply something isn't working right
